src/Service/Messenger.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Service;
+
+
+use App\Model\Message;
+
+class Messenger
+{
+    /**
+     * @var SenderInterface[]
+     */
+    protected $senders = [];
+
+    /**
+     * Messenger constructor.
+     * @param SenderInterface[] $senders
+     */
+    public function __construct(array $senders = [])
+    {
+        $this->senders = $senders;
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Service is full {@inheritDoc} but it still does not have any inheritance and parent methods usually do not contain any phpdocs too